microbench/src/main/java/alluxio/worker/BlockStoreReadBench.java

```diff
@@ -0,0 +1,274 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker;
+
+import alluxio.AlluxioTestDirectory;
+import alluxio.AlluxioURI;
+import alluxio.conf.Configuration;
+import alluxio.proto.dataserver.Protocol;
+import alluxio.underfs.UnderFileSystemConfiguration;
+import alluxio.util.io.PathUtils;
+import alluxio.worker.block.BlockStore;
+import alluxio.worker.block.CreateBlockOptions;
+import alluxio.worker.block.io.BlockReader;
+import alluxio.worker.block.io.BlockWriter;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.PooledByteBufAllocator;
+import io.netty.buffer.Unpooled;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+import org.openjdk.jmh.annotations.Warmup;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.nio.ByteBuffer;
+import java.util.Random;
+
+@Fork(value = 1, jvmArgsPrepend = "-server")
+@Warmup(iterations = 2, time = 3)
+@Measurement(iterations = 2, time = 3)
+@BenchmarkMode(Mode.Throughput)
+public class BlockStoreReadBench {
+  private static final int MAX_SIZE = 64 * 1024 * 1024;
+
+  /**
+   * A mock consumer of the data read from the store.
+   */
+  private static final byte[] SINK = new byte[MAX_SIZE];
+
+  @State(Scope.Benchmark)
+  public static class BenchParams {
+    private final Random mRandom = new Random();
+
+    @Param({"16", "64"})
+    public long mBlockSizeMB;
```

Comment:
For paged block store, we also want to see the impact of page sizes on read performance.